File: Estudiante.java
//Clase estudiante que implementa a la interfaz persona
public class Estudiante implements Persona{
    String ID;
    String Nombre;
    String CURP;
    String Domicilio;
    
    public Estudiante(String ID, String Nombre, String CURP, String Domicilio){
        this.ID = ID;
        this.Nombre = Nombre;
        this.CURP = CURP;
        this.Domicilio = Domicilio;
    }

    @Override
    public String getID() {
        return ID;
    }

    @Override
    public String getNombre() {
        return Nombre;
    }

    @Override
    public String getCURP() {
        return CURP;
    }

    @Override
    public String getDomicilio() {
        return Domicilio;
    }
}

File: Estudiante.py
from Persona import Persona

class Estudiante(Persona):
    def __init__(self,ID,Nombre,CURP,Domicilio):
        self.ID=ID
        self.Nombre=Nombre
        self.CURP=CURP
        self.Domicilio=Domicilio
    
    def getID(self):
        return self.ID
    
    def getNombre(self):
        return self.Nombre
    
    def getCURP(self):
        return self.CURP
    
    def getDomicilio(self): 
        return self.Domicilio
    
    